tailwind.config.ts

```diff
@@ -18,13 +18,16 @@ module.exports = {
     extend: {
       screens: {
         xs: "430px",
+        "2xl": "1400px",
```

Comment:
### **MUST NOT change default screen breakpoint values** !!
as it is used in other areas of the project and behavior on other areas will change. Use different names  for this or use the default value of 2xl and adjust your code according to that